main.go

```diff
@@ -358,9 +343,9 @@ func runPrep(commandString string) (*exec.Cmd, error) {
 */
 func startProcess(commandString string, theProcessType ProcessType) (*exec.Cmd, error) {
 	var err error
-	Info.log("Running: " + commandString)
+	ControllerInfo.log("Running command:  " + commandString)
```

Comment:
I think this log message should be moved down right before `cmd.Start()` is called.